### ISSUE TITLE
Failed to run single-file-cli with headers

### ISSUE THREAD

**Issue by yeger00** [gildas-lormeau/single-file-cli#87]
Hello,
I tried to run `single-file-cli` but failed to do. I keep getting the following error:
```
docker run singlefile --http-header "a=b" "https://www.wikipedia.org" > wikipedia.html


WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Cannot read properties of undefined (reading 'trim')
```

By looking at the code I see that it probably fail [here](https://github.com/gildas-lormeau/single-file-cli/blob/4230b50a0c1e05afc4da88f45fe1b893c4eebade/single-file-launcher.js#L82).

What am I missing?
Thanks for the help

Avi

**Comment by yeger00** [comment]
Figured it out I should do:
```
docker run singlefile --http-header="a=b" "https://www.wikipedia.org" > wikipedia.html
```

**Comment by gildas-lormeau** [comment]
Thank you, I confirm it was a bug. It's fixed in the version `2.0.38` that I've just published.

**Comment by yeger00** [comment]
Thanks @gildas-lormeau 
Much appreciated
